content/swiftui/concepts/viewmodifier/terms/animation/animation.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: '.animation()'
+Description: 'Applies an animation to a view.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.animation()`** modifier applies an animation to a [view](https://www.codecademy.com/resources/docs/swiftui/views).
+
+## Syntax
+
+```pseudo
+< View to be animated >
+    < View modifier that varies depending on switchValue >
+    .animation(animation, value: switchValue)
```

Comment:
```suggestion
View
    .animation(animation, value: switchValue)
```

Maybe include that comment on touches on what values `switchValue` can be (include that in the second bullet point) like:

"- A `value`, when updated, triggers the animation."